staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/capmanagers_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package fieldmanager_test
+package fieldmanager
```

Comment:
Why is this changing? (and others below).

Reply:
I don't think all the tests have to be in the same "package", I hope you can have external and internal tests in the same directory.

Reply:
`fieldmanager_test.go` creates a new version converter (private function) so that had to be changed to an internal test. The rest of the packages transitively depend on a function in the field manager test file and had to be updated as well. I did try to change the minimal set of files (only changed if a test was failing) so there should still be some external tests in this directory.

Reply:
Ah, good point, thanks for clarifying, I definitely missed that. There are lot more things that we can improve here I think, but let's block on this for now.